geometry_msgs/Point curr
geometry_msgs/Point dest
---
